### ISSUE TITLE
Missing lang attribute on html tag

### ISSUE THREAD

**Issue by btzr-io** [lbryio/lbry-desktop#6483]
**Bug**

> Identifying the language of the page or page elements allows screen readers to read the content in the appropriate language. It also facilitates automatic translation of content.

**Comment by btzr-io** [comment]
Fix https://github.com/lbryio/lbry-desktop/pull/6470/commits/d0e865b9ec99433268024433b3982ac5e4ae6607
